kernel/kernel-websubclient-api/src/main/java/io/mosip/kernel/websub/api/client/SubscriberClientImpl.java

```diff
@@ -151,4 +165,47 @@ public SubscriptionChangeResponse unSubscribe(UnsubscriptionRequest unsubscripti
 		}
 	}
 
+
+
+
+	@Override
+	public FailedContentResponse getFailedContent(FailedContentRequest failedContentRequest) {
+		HttpHeaders headers = new HttpHeaders();
+		headers.set("Accept", MediaType.APPLICATION_JSON_VALUE);
+		headers.set(WebSubClientConstants.SUBSCRIBER_SIGNATURE_HEADER, getHmac256(failedContentRequest.getTopic()+failedContentRequest.getCallbackURL()+failedContentRequest.getTimestamp(),failedContentRequest.getSecret())); 
```

Comment:
Can we include message count also if not null into the HMAC?

Reply:
sure